pac-aurora-provisioner/README.md

```diff
@@ -47,6 +47,39 @@ docker run \
 
 Note that the process may take approximately an hour to provision fully.
 
+## Provisioning a cluster from an existing database snapshot
+
+The provisioning process will reflect the same for provisioning a cluster with an empty database
+except that the Aurora cluster in eu-west-1 will be created from an existing DB snapshot in AWS.
```

Comment:
Presumably this includes configuring cross-region replication, so the restored snapshot will be propagated transparently to the other region? I think it's worth being clear about that - it's not something we'd want to fret over if we had to do this "for real".

Reply:
Yes, It is.